CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 The overarching project is guided by [https://catena-x.net](https://catena-x.net) and all development work is covered in [https://eclipse-tractusx.github.io](https://eclipse-tractusx.github.io/).
 
+The official changelog, including compatibility matrices for Tractus-X releases, can be found at [https://eclipse-tractusx.github.io/changelog](https://eclipse-tractusx.github.io/changelog).
+
```

Comment:
@ds-mmaul Thanks for the PR!
Just a proposal for a small rephrasing:
The overarching changelog for Tractus-X releases, including compatibility matrices, can be found at [https://eclipse-tractusx.github.io/changelog](https://eclipse-tractusx.github.io/changelog).

I think @Siegfriedk should decide whether to place this sentence into the changelog of this repo or directly into the TRG 1.03. I think he uses the changelog of this repo as base for what ends up on the [Tractus-X website](https://eclipse-tractusx.github.io/CHANGELOG), and I'd like to avoid this sentence ending up there, because it would reference the same page.

Reply:
I actually updated the release process last release here https://github.com/eclipse-tractusx/tractus-x-release/blob/main/RELEASE.md

Basically i create the Changelog in this repository and copy&paste it after the release to the eclipse tractus-x website and to the EF automotive.x tractus-x website.

The Changelog will therefore always be correct and if i don't understand it right, the sentence wouldn't make sense as the source of truth is this tractus-x-release repository and not the website.

Reply:
@evegufy sure, will change the sentence

The reason why I put it in here initially is only because of the example changelog reference in TRG 1.03.

Reply:
Btw. we have been revisiting nearly all TRGs and created tickets which are updating/finetuning them.

One ticket https://github.com/eclipse-tractusx/sig-infra/issues/363 is about the Changelog. We want to add a Tips&Tricks section to more TRGs and for the Changelog we could optimize the example changelog.

The example changelog would be easy / quick to update as it doesn't state that you have to use that template it only shows a good example.

Reply:
@Siegfriedk sounds like a good way to me. How about we just update the issue with an additional AC:
provide an example changelog link that includes 

`The overarching changelog for Tractus-X releases, including compatibility matrices, can be found at https://eclipse-tractusx.github.io/changelog.`

Reply:
We can add this hint. We also should add both references, the website and the tractus-x-release repository. Teh repository is the source of truth

Reply:
I have updated the ticket: https://github.com/eclipse-tractusx/sig-infra/issues/363

Reply:
Thank you, @Siegfriedk, with that, I would close this PR and discussion